src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -72,14 +74,11 @@ def _expand_mask(mask: torch.Tensor, dtype: torch.dtype, tgt_len: int = None):
     """
     Expands attention_mask from `[bsz, seq_len]` to `[bsz, 1, tgt_seq_len, src_seq_len]`.
     """
-    batch_size, source_length = mask.size()
+    batch_size, source_length = mask.shape
     tgt_len = tgt_len if tgt_len is not None else source_length
 
-    expanded_mask = mask[:, None, None, :].expand(batch_size, 1, tgt_len, source_length).to(dtype)
-
-    inverted_mask = 1.0 - expanded_mask
-
-    return inverted_mask.masked_fill(inverted_mask.to(torch.bool), torch.finfo(dtype).min)
+    expanded_mask = mask[:, None, None, :].to(dtype).expand(batch_size, 1, tgt_len, source_length)
+    return ~expanded_mask
```

Comment:
This will only work if `dtype=bool`, so either reverted to `1.0-expanded_mask` or don't accept any dtype here.